proposal.md

```diff
@@ -2003,12 +2003,16 @@ Example: `"cannot use swap() with boolean"`.
 
 *Complexity:* Constant.
 
+##### Add element to array
+
 ```cpp
 void push_back(basic_json &&);
 void push_back(const basic_json &);
 void push_back(const typename object_type::value_type &);
 ```
 
+[TODO: Discuss. The third overload uses `object_type::value_type` which seems strange in the context of adding to an array.]

```

Comment:
Yes, silly me. Most probably I missed to specify this particular overload in the context of object, silly me. Should be separated and specified accordingly.